textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/connector/OneToNBroadcastConnector.java

```diff
@@ -17,14 +17,18 @@
  */
 public class OneToNBroadcastConnector implements IConnector {
     
+    private static final int PRE_OPEN = -2;
+    private static final int OPENED = 0;
```

Comment:
In IOperator's interface, we have defined "-1" for closed and "0" for opened. So my intention was to be consistent with other operators.